QATCH/processors/Analyze.py

```
@@ -6733,7 +6729,8 @@ def monoCurve(x, a, b, c, d):
 
             np.asarray(t_minima)
 
-            ax2.plot(xs[zeros3[0]:], ys_diss_diff_offset[zeros3[0]:], "b:")
+            l = min(len(xs), len(ys_diss_diff_offset))
```

Ruff (0.15.2)

[error] Ambiguous variable name: `l`

(E741)